tests/test_attribute_definition.py

```diff
@@ -3,24 +3,25 @@
 """
 
 from __future__ import annotations
+
 from dataclasses import dataclass
 from io import StringIO
-from typing import Any, TypeVar, cast, Annotated, Generic, TypeAlias
+from typing import Annotated, Any, Generic, TypeAlias, TypeVar, cast
 
 import pytest
-from xdsl.dialects.builtin import IndexType, IntegerAttr, IntegerType, Signedness
 
+from xdsl.dialects.builtin import IndexType, IntegerAttr, IntegerType, Signedness
 from xdsl.ir import Attribute, Data, ParametrizedAttribute
 from xdsl.irdl import (
+    AnyAttr,
     AttrConstraint,
+    BaseAttr,
     ConstraintVar,
     GenericData,
+    ParamAttrDef,
     ParameterDef,
     irdl_attr_definition,
     irdl_to_attr_constraint,
-    AnyAttr,
-    BaseAttr,
-    ParamAttrDef,
 )
 from xdsl.parser import AttrParser
 from xdsl.printer import Printer
```

Comment:
These changes seem unrelated. Can you remove them? (check out `git add -p` to add parts of files)

Reply:
Addressed in #1243.